frontend/src/Views/MainPage/MainPage.tsx

```diff
@@ -62,15 +62,31 @@ const MainPage: React.FC = () => {
    * @param     moduleID: string => An InputModule ID to be searched and updated
    * @param     nextModuleID: string => A second InputModule ID to be searched and updated
    */
-  const handleFormState = (moduleID: string, nexModuleID?: string, prevModuleID?: string) => {
+  const handleFormState = (formMovement?: string) => {
+    const moduleID = formStates[currForm].id;
+    let secondModuleID = null;
+    switch (formMovement) {
+      case 'back':
+        secondModuleID = formStates[currForm - 1].id;
+        setCurrForm((prevForm) => prevForm - 1);
+        break;
+      case 'forward':
+        secondModuleID = formStates[currForm + 1].id;
+        setCurrForm((prevForm) => prevForm + 1);
+        break;
+      default:
+        return;
+    }
+    // if (moveBack) {
+    //   secondModuleID = formStates[currForm - 1].id;
+    //   setCurrForm((prevForm) => prevForm - 1);
+    // }
+    // if (moveForward) {
+    //   secondModuleID = formStates[currForm + 1].id;
+    //   setCurrForm((prevForm) => prevForm + 1);
+    // }
```

Comment:
We forgot to remove this

Reply:
This is outdated, we cleaned it yesterday. 😛